### ISSUE TITLE
Restore Safari COLOR icons in Toolbar?

### ISSUE THREAD

**Issue by Thannah** [w0lfschild/cDock#8]
Any chance you can add a feature to restore the Safari COLOR icons in its Toolbar?

I am going blind trying to distinguish one from the other with these thin-lines B&W icons!

**Comment by w0lfschild** [comment]
Make sure you're using the latest version of cDock.

**Comment by Thannah** [comment]
Thank you for your reply, Wolfgang.

I am using version 8.1 of cDock.

It works great in the Dock.

But I am wondering whether you can restore the color icons in Safari’s toolbar. Since Yosemite, the new thin, tiny black and white icons in Safari’s Toolbar tell me nothing about their function and are ugly!

Can you possibly apply your genius fix this too?

Thanks,

Nate

**Comment by w0lfschild** [comment]
I suppose I'm not even sure what you're talking about... Safari's toolbar has never been colored and I don't even use Safari because it sucks.

What does Safari's toolbar have to do with the Dock?

You should be asking this person:
http://hetima.com/safaristand/ 
@hetima

**Comment by Thannah** [comment]
Dear Wolfgang,

I’m very sorry for any confusion. I never used Safari before FireFox started crashing with latest version, so I didn’t know it was always this ugly and cryptic.

When I saw tiny black and white icons in the Safari toolbar, I thought this was part of the new ugly minimalist theme of Yosemite, just like the new black and white icons in the Yosemite menubar. I did not know that those Safari icons were always that ugly and always only black and white!

I will try to contact the SafariStand author as you suggest.

Thanks for cDock - it is wonderful!

Best wishes, Nate :D
